packages/web/src/components/avatar/Avatar.tsx

```diff
@@ -0,0 +1,65 @@
+import {
+  ID,
+  Maybe,
+  SquareSizes,
+  accountSelectors,
+  cacheUsersSelectors
+} from '@audius/common'
+import { Link } from 'react-router-dom'
+
+import DynamicImage from 'components/dynamic-image/DynamicImage'
+import { useUserProfilePicture2 } from 'hooks/useUserProfilePicture'
+import { useSelector } from 'utils/reducer'
+import { SIGN_IN_PAGE, profilePage } from 'utils/route'
+
+import styles from './Avatar.module.css'
+
+const { getAccountUser } = accountSelectors
+
+const { getUser } = cacheUsersSelectors
+
+const messages = {
+  goTo: 'Go to',
```

Comment:
is `go to` std verbiage?

Reply:
From chat gpt perspective yeah, but can dig into whether there is better verbiage